functional-test-app/build.gradle

```diff
@@ -5,9 +5,9 @@ buildscript {
     }
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
-        classpath "com.bertramlabs.plugins:asset-pipeline-gradle:3.0.7"
-        classpath "org.grails.plugins:hibernate5:${gormVersion-".RELEASE"}"
-        classpath "gradle.plugin.com.energizedwork.webdriver-binaries:webdriver-binaries-gradle-plugin:1.4"
+        classpath "com.bertramlabs.plugins:asset-pipeline-gradle:5.0.1"
+        classpath "org.grails.plugins:hibernate5:$gormVersion"
+        classpath "gradle.plugin.com.github.erdi.webdriver-binaries:webdriver-binaries-gradle-plugin:2.7"
```

Comment:
Converting to ContainerGebSpec removes the requirement for this.  Going forward, the webdriver binaries are abandoned so I would switch to the newer way.  You can also remove the withTest, GebConfig, web driver config, and any Geb headless properties in the CI.  The geb plugin has an updated readme that shows a single gradle line to support ContainerGebSpec.